07/ID_funkce/ID_format.py

```diff
@@ -0,0 +1,14 @@
+def ID_format(ID):
+
+    first_six_numbers = ID[:6]
+    slash = ID[6]
+    last_four_numbers = ID[7:]
+
+    if (first_six_numbers.isdigit() and len(first_six_numbers) == 6
```

Comment:
`len` na prvých 6 nie je potreba kontrolovať, pretože to naozaj vezme prvých 6 znakov (index `:6`). Kontrola dĺžky je u premennej `last_four_numbers` potreba preto lebo `7:` ti zo stringu vezmem všetko čo je od 8. znaku do konca a tam to záleží na dĺžke toho stringu, takže to môže byť hocičo.